lib/menu.rb

```diff
@@ -0,0 +1,15 @@
+require_relative 'menulist.rb'
+class Menu
+include MenuList
+
+attr_reader :menu
+  def initialize (menu_type)
```

Comment:
Style/SpaceAfterMethodName: Do not put a space between a method name and the opening parenthesis. (https://github.com/bbatsov/ruby-style-guide#parens-no-spaces)